sql/core/src/test/resources/sql-tests/results/inner-join.sql.out

```diff
@@ -0,0 +1,68 @@
+-- Automatically generated by SQLQueryTestSuite
+-- Number of queries: 13
```

Comment:
Actually, this number is wrong. Next time, please do not manually change this file. You should run the command to generate the file. @stanzhai